Development.md

```diff
@@ -0,0 +1,26 @@
+# Development documentation
+
+## Writing unit tests
+
+### Tests that compare with OpenCV
+
+[OpenCV](https://opencv.org/) is a popular image processing library for C++ and Python.
+We use it as a reference for some of the functions implemented in ImageJS.
+
+All images used for comparison with OpenCV are generated by the Python script [`test/img/opencv/generate.py`](./test/img/opencv/generate.py).
+
+To add a new test reference file:
+
+- Update [`generate.py`](./test/img/opencv/generate.py) with the OpenCV code that creates the file.
+- Run `generate.py` (see following paragraph).
+- Add the new filename to the [`TestImagePath`](./test/TestImagePath.ts) type (alphabetical order).
+
+To run the generation script, use the following steps:
+
+- Install Python 3.x
+- Run `python -m venv .venv` to create a virtual environment for the project
```

Comment:
on mac with `brew install python`
`python` bin does not exists, we must use `python3`

Once the `venv` is active, `python` is working